test/quantization/core/test_quantized_tensor.py

```diff
@@ -904,16 +885,16 @@ def test_qtensor_reshape(self):
 
     def test_qtensor_unsqueeze(self):
         for device in get_supported_device_types():
-            x = torch.randn((1, 3, 4))
-            qx = torch.quantize_per_tensor(x.to(device), scale=1.0, zero_point=0, dtype=torch.quint8)
+            x = torch.randn((1, 3, 4)).to(device)
```

Comment:
This is not idiomatic; better is `torch.randn((1, 3, 4), device=device)` which will skip initializing the tensor on CPU and then copying it to CUDA

Reply:
alright, will fix